Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-FIPFSScanEngineVersionState.ps1

```diff
@@ -0,0 +1,147 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\..\..\..\Shared\Invoke-ScriptBlockHandler.ps1
+
+Function Get-FIPFSScanEngineVersionState {
+    [CmdletBinding()]
+    [OutputType("System.Bool")]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]
+        $ComputerName,
+        [Parameter(Mandatory = $false)]
+        [scriptblock]
+        $CatchActionFunction
+    )
+
+    begin {
+        Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+        function GetItemFromExchangeInstallPath {
+            param(
+                [string]$ExchangeSubDir
+            )
+
+            Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+
+            $finalPathToQuery = Join-Path $env:ExchangeInstallPath $ExchangeSubDir
+            $getItem = Get-Item -Path $finalPathToQuery
+
+            $returnObject = ([PSCustomObject]@{
```

Comment:
Just do a `return` vs saving to a variable. No need for the extra lines

Reply:
Will be fixed with the upcoming commit.